CHANGELOG.md

```diff
@@ -1,15 +1,25 @@
 ## [Unreleased]
+### Added
+ - Prep work for monorepo support. #TINY-6986, #TINY-6987
+
+### Changed
+ - Now tags as `vX.Y.Z` instead of `module@vX.Y.Z`. The latter form will be used for non-primary modules of monorepos. #TINY-7026
```

Comment:
Should this be `X.Y.Z` instead of `vX.Y.Z` given the note in the beehive channel earlier?

Reply:
Ah. Missed that one. Ta.